### PR TITLE
[OTA-3961] Omniture Analytics configuration

### PR DESCRIPTION
**Pull request by piotrekc**
Signed-off-by: Piotr Cichon <ext-piotr.cichon@here.com>

### CODE REVIEW

File: ota-plus-web/app/com/advancedtelematic/controllers/Application.scala [hunk 1]
```diff
@@ -66,6 +83,7 @@ class Application @Inject() (ws: TraceWSClient,
 
   private[this] val oauthConfig = OAuthConfig(conf)
   private[this] val uiToggles = UiToggles(conf)
+  private[this] val environmentMode = EnvironmentMode(env)
```
Comment (houcros) [inline ota-plus-web/app/com/advancedtelematic/controllers/Application.scala]: We don't use this, why do we need it?

Comment (houcros) [reply ota-plus-web/app/com/advancedtelematic/controllers/Application.scala]: My bad, we do use it.

File: ota-plus-web/app/com/advancedtelematic/controllers/Application.scala [hunk 2]
```diff
@@ -45,6 +47,20 @@ object UiToggles {
   }
 }
 
+final case class EnvironmentMode(
```
Comment (houcros) [inline ota-plus-web/app/com/advancedtelematic/controllers/Application.scala]: Looks fine, I just find it slightly easier to read as:
```
final case class EnvironmentMode(isDev: Boolean, isProd: Boolean)
object EnvironmentMode {
  def apply(env: Environment): EnvironmentMode =
    EnvironmentMode(env.mode == Dev, env.mode == Prod)
}
```
In any case, we should probably move this to `Data.scala` in this package and import it here, to try to keep all the models in one place.

Comment (houcros) [reply ota-plus-web/app/com/advancedtelematic/controllers/Application.scala]: Also: good use of the companion object 👌

File: ota-plus-web/app/com/advancedtelematic/controllers/Application.scala (hunk 1 above)
Comment (piotrekc) [reply ota-plus-web/app/com/advancedtelematic/controllers/Application.scala]: Okay

File: ota-plus-web/app/com/advancedtelematic/controllers/Application.scala (hunk 2 above)
Comment (piotrekc) [reply ota-plus-web/app/com/advancedtelematic/controllers/Application.scala]: Done, thanks

**Comment by piotrekc** [comment]
> LGTM, just one minor comment.
> Also it's not obvious to me how the `Environment` will be set to `Prod` or `Dev`. Will it be set automatically by Play? Or do we need to do anything else?

I think it should be handled by playframework by running the created production build.

**Pull request merged**